triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Message.scala

```diff
@@ -14,6 +14,8 @@ final case class StartFailed(cause: Throwable) extends Message
 
 final case class Started(binding: ServerBinding) extends Message
 
+case object Stop extends Message
```

Comment:
Does intellij wants things ordered alphabetically? Previously it was somewhat ordered by the stages in which the message can be received. Just curious, I don’t care about the ordering.

Reply:
I tried to make the ordering "nicer" when I did a previous refactoring, but didn't realise that this one was intentionally grouped with other Server messages (as opposed to TriggerRunner messages). There should probably be a comment in this file pointing out the two groups.